src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -20,90 +20,177 @@ public SuperStructure(
     this.armSensor = armSensor;
   }
 
-  public Command levelFour(BooleanSupplier score) {
+  public Command coralLevelFour(BooleanSupplier score) {
     return Commands.sequence(
-        Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_FOUR_PRE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_UP),
-            spinnyClaw.stop()),
-        armPivot.moveToPosition(ArmPivot.PRESET_PRE_L4),
-        Commands.waitUntil(score),
-        Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_FOUR_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_L4)),
-        spinnyClaw.holdExtakePower().withTimeout(0.2),
-        preIntake());
+            Commands.parallel(
+                elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_FOUR_PRE_POS),
+                armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP),
+                spinnyClaw.stop()),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_PRE_L4),
+            Commands.waitUntil(score),
+            Commands.parallel(
+                elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_FOUR_POS),
+                armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L4)),
+            spinnyClaw.coralHoldExtakePower().withTimeout(0.2),
+            preIntake())
+        .withName("Coral Level 4");
   }
 
-  public Command levelThree(BooleanSupplier score) {
+  public Command coralLevelThree(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_THREE_PRE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_UP),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_THREE_PRE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP),
             spinnyClaw.stop()),
-        armPivot.moveToPosition(ArmPivot.PRESET_L2_L3),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L2_L3),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_THREE_POS),
-        spinnyClaw.holdExtakePower().withTimeout(0.15),
-        preIntake());
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_THREE_POS),
+        spinnyClaw.coralHoldExtakePower().withTimeout(0.15),
+        preIntake().withName("Coral Level 3"));
```

Comment:
```suggestion
        preIntake()).withName("Coral Level 3");
```
This makes the `.withName()` apply to the entire command, not just the `preIntake()` command.